web/src/components/NavbarTop.vue

```diff
@@ -25,6 +25,12 @@ const route = useRoute()
         :to="{ name: 'caselaw' }"
         >Rechtsprechung</router-link
       >
+      <router-link
+        class="hover:bg-yellow-500 hover:underline p-8"
+        :class="{ underline: route.path.includes('norms') }"
+        :to="{ name: 'norms' }"
+        >Normen</router-link
+      >
```

Comment:
Nitpick:
This is a repeating pattern with duplicated code. The purpose of the component is render a list of links. Each of these links should have the same look and effects. At least once a third link gets added, we should refactor this. Till then, it might be fine to just keep it like that. Feel free to decide on your own. 🙃 
With refactoring I mean to have a in the setup function a static list like `[{ label: "Rechtsprechung", routeName: "caselaw" }, {...}]` and then a `v-for` loop in the template. Just as a recommendation. This would also cleanly separate data and logic. 🙃